src/Behat/Testwork/Tester/Exception/WrongPathsException.php

```diff
@@ -20,30 +20,30 @@
 final class WrongPathsException extends RuntimeException implements TesterException
 {
     /**
-     * @var string
+     * @var string[]
      */
-    private $path;
+    private $paths;
 
     /**
      * Initializes exception.
      *
-     * @param string $message
-     * @param string $path
+     * @param string   $message
+     * @param string[] $paths
      */
-    public function __construct($message, $path)
+    public function __construct($message, $paths)
     {
         parent::__construct($message);
 
-        $this->path = $path;
+        $this->paths = $paths;
```

Comment:
For BC please ensure that `$paths` argument can be either a string or an array. Both must be supported for now.

```
$this->paths = (array) $paths;
```